game/static/game/css/board.css

```
@@ -1169,12 +1170,14 @@ body {
     text-overflow: ellipsis;
     white-space: nowrap;
     min-width: 0;
+    flex: 1 1 auto;
 }
 
 .clock .time {
     font-family: 'Consolas', 'Courier New', monospace;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Consolas" (font-family-name-quotes)

(font-family-name-quotes)